src/test/ui/privacy/access_levels.stderr

```diff
@@ -0,0 +1,122 @@
+error: access lvl: None, Public vis: pub(access_levels), Exported vis: pub(access_levels), Reachable vis: pub(access_levels), ReachableFromImplTrait vis: None
+  --> $DIR/access_levels.rs:4:1
+   |
+LL | mod outer {
+   | ^^^^^^^^^
+
+error: access lvl: Some(Exported), Public vis: pub(access_levels), Exported vis: Public, Reachable vis: Public, ReachableFromImplTrait vis: None
```

Comment:
```suggestion
error: effective visibility: Public: pub(access_levels), Exported: Public, Reachable: Public, ReachableFromImplTrait: Public
```
- the `Some(Exported)` part is redundant (although it may be convenient for testing)
- the "vis" parts only create visual noise
- `ReachableFromImplTrait` should also be set to `Public`

Reply:
Or slightly more compact:
```suggestion
error: effective visibility: Public: access_levels, Exported: pub, Reachable: pub, ReachableFromImplTrait: pub
```